subjects/throttle/README.md

```diff
@@ -11,3 +11,5 @@ Create two functions that will work like `_.throttle` from lodash
 ### Notions
 
 - [lodash.com/docs/4.17.15#throttle](https://lodash.com/docs/4.17.15#throttle)
+- [https://css-tricks.com/debouncing-throttling-explained-examples/#throttle](https://css-tricks.com/debouncing-throttling-explained-examples/#throttle)
+- [https://stackoverflow.com/questions/24079736/confused-about-the-maxwait-option-for-lodashs-debounce-method](https://stackoverflow.com/questions/24079736/confused-about-the-maxwait-option-for-lodashs-debounce-method)
```

Comment:
I'm not sure about this. We agreed that the link put in notions are specific links, like "devdocs.io" or "lodash". I think that if students search for it they will be able to find it, what do you think?

Reply:
https://github.com/01-edu/public/tree/master/subjects/sweet-curry has link in Notions to stackoverflow. I think it will be useful.